### ISSUE TITLE
Inculde FAQ

### ISSUE THREAD

**Issue by Patschke** [raithos/xwing#343]
While it is pretty clear, that we simply add errata to the card text (leaving a not that an errata changed the text), I'd also like to add the FAQ. There are two sources: 
* https://community.fantasyflightgames.com/topic/277390-x-wing-official-rulings/
* https://images-cdn.fantasyflightgames.com/filer_public/2a/57/2a57fdb2-e13f-4d58-b9db-bb082dd23dcc/swzrulesreference_v102.pdf (last page)

Most of those affect a certain set of cards, or common interaction between cards. So I thought it could be usefull, to display those FAQ when a squad is affected by them (e.g. if a squad contains Sabine Wren and Mines, Outmaneuver on a ship that has no Front-Arc primary attack,... ). I am not sure where to add that, and whether to add it to the printed lists as well. 

One could even consider adding the rules reference in a dedicated tab (e.g. between factions and card browser). It could maybe look somewhat like the browser; having a drop down on the left containing all the key words, and the text on the right. Maybe it should have something like a next and previous button as well. If we add something like this, we could make all occurences of phrases that are described in the rules references clickable, so one could e.g. click "fully execute" on the TIE Defender ship ability, to open the rules reference for "fully execute". This would require some more work than just the above suggestion, but would also be a cool thing to have.

**Comment by raithos** [comment]
let me think about this for a bit, i agree in principle, i just need to wrap my head around with a bit more

**Comment by Patschke** [comment]
...?

**Comment by raithos** [comment]
So I have been thinking about it for some time... 

If we limit our scope to a utility tool, I think your FAQ tab could avoid some of that, specifically for making it easier to access the contents of the pdf with links to the latest files for people to reference them, and a quick search function for the game terms would be good. I'd be ok with that as we'd be just making that resource more easily accessible, especially for quick reference during tournaments.

I'm not a fan of adding it built into the listbuilder, I think that will be both very troublesome to manage as the games complexity grows. I'm also not that comfortable with integrating because it will indirectly suggest rulings on unintended interactions which is not our job.. I know there are already people who use YASB as the be all resource for their card text, and when we make mistakes people are mislead. While that is pretty easily remedied because there are cards to correct them, it gets a lot more murky when it comes to rules as there's a fair bit of interpretation and judges have this issue all the time. If we stick our necks out and also reference rules, I am confident people will start citing our references as reasons why a rule is right or wrong as they do with card text already, and I'd rather stay out of that.

**Comment by raithos** [comment]
TLDR yes to FAQ tab for reference, no to in builder references

**Comment by raithos** [comment]
I finally got around to working on this. its now implemented. TOOK FOREVER
